requirements-docs.txt

```diff
@@ -5,4 +5,4 @@ PyYAML==3.11
 Pillow==2.6.1
 aexpect==1.0.0
 # stevedore for loading "new style" plugins
-stevedore==1.8.0
```

Comment:
Actually as 1.8 does not work, shouldn't we increase the `requirements.txt` as well? (that is what I was experimenting with before I had to leave)